File: Python/program/Clothing.py
# Import Class Induk
from Product import Product

# Deklarasi Class

class Clothing(Product):

    # Privat (__), Protected (_), Public Atributes
    __size = ""
    __material = ""
    __gender = ""

    # Konstruktor
    def __init__(self, size = "", material = "", gender = ""):
        self.__size = size
        self.__material = material
        self.__gender = gender

    # Getter dan Setter
        
    # Get size
    def get_size(self):
        return self.__size
    
    # Set size
    def set_size(self, size):
        self.__size = size

    # Get material
    def get_material(self):
        return self.__material
    
    # Set material
    def set_material(self, material):
        self.__material = material

    # Get gender
    def get_gender(self):
        return self.__gender
    
    # Set gender
    def set_gender(self, gender):
        self.__gender = gender

File: PHP/program/Clothing.php

<?php
require ('Product.php');

// Deklarasi Class
class Clothing extends Product{
    private $size;
    private $material;
    private $gender;

    // Konstruktor
    public function __construct($material, $size, $gender) {
        parent::__construct($idProduct, $name, $brand, $price); // Call the constructor of the parent class
        $this->material = $material;
        $this->size = $size;
        $this->gender = $gender;
    }

    // Getter dan Setter
    
    // Get material
    public function get_material() {
        return $this->material;
    }

    // Set material
    public function set_material($material) {
        $this->material = $material;
    }

    // Get size
    public function get_size() {
        return $this->size;
    }

    // Set size
    public function set_size($size) {
        $this->size = $size;
    }

    // Get gender
    public function get_gender() {
        return $this->gender;
    }

    // Set gender
    public function set_gender($gender) {
        $this->gender = $gender;
    }
}